app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/BgSendQueue.java

```diff
@@ -39,6 +40,10 @@
     @Column(name = "operation_type")
     public String operation_type;
 
+    private static Context mContext = Home.getContext();
```

Comment:
Yep.  I can test out the changes.  If it isn't too big a hassle for you.
If you get it done (your) tonight, I can test it later (my) today ;).

On Wed, May 20, 2015 at 10:30 AM, Stephen Black <notifications@github.com>
wrote:

> In
> app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/BgSendQueue.java
> <https://github.com/StephenBlackWasAlreadyTaken/xDrip-Experimental/pull/23#discussion_r30662297>
> :
>
> > @@ -39,6 +40,10 @@
> >      @Column(name = "operation_type")
> >      public String operation_type;
> >
> > +    private static Context mContext = Home.getContext();
>
> I can change it over tonight if you can test out the changes tomorrow!
> … <#14d6ebae2a994097_>
> On May 19, 2015 8:29 PM, "John" <notifications@github.com> wrote: In
> app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/BgSendQueue.java
> <
> https://github.com/StephenBlackWasAlreadyTaken/xDrip-Experimental/pull/23#discussion_r30662228>
> : > @@ -39,6 +40,10 @@ > @Column <https://github.com/Column>(name =
> "operation_type") > public String operation_type; > > + private static
> Context mContext = Home.getContext(); Ok. Good feedback. I would prefer it
> as a service myself. Would it be ok to leave as is for now, and I make it a
> service over time? I want to replace the offline watch face really bad, but
> don't have much time right now. Also waiting for my new pebble in June.
> John Stevens M - +61 418526349 … <#14d6eb96d34569bf_> On 20 May 2015, at
> 07:56, Stephen Black <notifications@github.com> wrote: In
> app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/BgSendQueue.java:
> > @@ -39,6 +40,10 @@ > @Column <https://github.com/Column> <
> https://github.com/Column>(name = "operation_type") > public String
> operation_type; > > + private static Context mContext = Home.getContext();
> gotcha gotcha (and yeah, sorry github is also awful on phones) If we are
> worried about creating too many we should either have it be a singleton or
> a service, either would be fine with me. Im more of a fan of services as
> they feel more android-ish but they seem to have more resource overhead —
> Reply to this email directly or view it on GitHub. — Reply to this email
> directly or view it on GitHub <
> https://github.com/StephenBlackWasAlreadyTaken/xDrip-Experimental/pull/23/files#r30662228>
> .
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/StephenBlackWasAlreadyTaken/xDrip-Experimental/pull/23/files#r30662297>
> .
>



-- 
John Stevens
"You are how you live, not what you have."